Documentation/doc/Documentation/Installation.txt

```diff
@@ -334,6 +336,24 @@ cmake -DCGAL_HEADER_ONLY=ON . # configure \cgal
 
 and we do not need to run `make` anymore.
 
+\subsubsection subsection_headeronly_without_configuration Header-only without CMake configuration
+
+Since \cgal 4.12, \cgal can be used in header-only mode, without even
+configuring \cgal libraries. In this case, the program using \cgal
+(example, test, demo, etc.) must be directly configured using CMake. The
```

Comment:
the program needs to be directly configured using cmake in any case. The new thing is that CGAL will be configured at the same time.